packages/meditrak-server/src/routes/exportSurveyResponses.js

```diff
@@ -285,6 +245,9 @@ export async function exportSurveyResponses(req, res) {
         });
       });
 
+      // Add export date and origin
+      exportData = addExportedDateAndOriginAtTheSheetBottom(exportData);
+
```

Comment:
```suggestion
      exportData = addExportedDateAndOriginAtTheSheetBottom(exportData);
exportData = [
        ...exportData,
        [],
        [],
        [`Data exported from Tupaia.org on ${moment().format('Do MMM YYYY')}`],
      ];
```

Reply:
This is an optional suggestion: I would just add the date and origin inline here and remove the util file. As I see it the advantages to this are:
- the 'addExportedDateAndOriginAtTheSheetBottom' function is very specific and is not really a utility that will be used anywhere else
- it is a simple addition that is easier to read inline
- overall this export code is 'procedural' so it is in general one off code that is written inline
- less overall code to maintain